src/components/layout/nav/nav-jump-indices.tsx

```diff
@@ -97,13 +94,9 @@ const JumpIndiceNav = ({ is_ppc }) => {
                     </LogoWrapper>
                 </ContentContainer>
             </Section>
-            <CFDWarning is_ppc={is_ppc} />
+            <CFDWarning {...props} />
```

Comment:
the spread props pattern will make the code unclear a little bit. why do we need this? this one is not recommended